javalib/src/main/scala/java/lang/ProcessHandle.scala

```diff
@@ -18,10 +18,6 @@ trait ProcessHandle {
 
   def destroyForcibly(): scala.Boolean
 
```

Comment:
In the JDK 23 documentation, `equals()` is described as being implemented (overridden)
in the Interface, at least to my interpret ion of the coffee grounds.  I believe that
this is because the JDK implementation does not use Object.equals() but implemts
a more complex test. More when I get to that section of the code below.

Where `equals()` goes, so goes `hashCode()`.